app/src/main/java/com/main/mealplanner/RetrofitClientInstance.kt

```diff
@@ -4,18 +4,18 @@ import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
-    private var retrofit: Retrofit? = null;
+    private var retrofit: Retrofit? = null
```

Comment:
semicolon not needed in Kotlin

Reply:
Good suggestion... but someone else beat you to this one.  It has already been listed on another code review.